client/components/indicators/index.js

```diff
@@ -19,20 +19,20 @@ const renderLastUpdated = ( lastUpdated ) => {
 	}
 
 	return (
-		<div className="form-setting-explanation indicator__last-updated">
+		<div className="form-setting-explanation indicators__indicator-last-updated">
```

Comment:
Why do we need `form-setting-explanation` here? Why doesn't it show a linting error?